lms/templates/main.html

```diff
@@ -76,9 +76,15 @@
   <%static:css group='style-app-extend1'/>
   <%static:css group='style-app-extend2'/>
 
+  <script type="text/javascript" src="${static.url("require-config-lms.js")}"></script>
```

Comment:
Why was this moved out of the "if enable_require_js" section (same question for line 95)?